BraintreeASPExampleTests/packages.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Braintree" version="2.59.0" targetFramework="net45" />
-  <package id="Moq" version="4.2.1510.2205" targetFramework="net45" />
+  <package id="Braintree" version="3.3.0" targetFramework="net452" />
+  <package id="Castle.Core" version="3.3.3" targetFramework="net452" />
```

Comment:
Do we need to reference this dependency in the section of the changed README below?